e2e/test/transactions.js

```diff
@@ -27,14 +28,34 @@ describe('transactions', () => {
     assert(toBN(balance).isZero(), 'Account should not have tokens yet')
 
     // send transaction to home chain
-    await homeWeb3.eth.sendTransaction({
+    const depositTx = await homeWeb3.eth.sendTransaction({
       from: user.address,
       to: HOME_BRIDGE_ADDRESS,
       gasPrice: '1',
-      gasLimit: '50000',
+      gas: '50000',
       value: '1000000000000000000'
     })
 
+    // Send a Tx to generate a new block
+    await generateNewBlockWithTx(homeWeb3, user.address)
+
+    // wait to send a Tx to generate a new block
+    await promiseRetry(
+      async retry => {
+        const lastBlockNumber = await homeWeb3.eth.getBlockNumber()
+        if (lastBlockNumber === depositTx.blockNumber + 2) {
+          await generateNewBlockWithTx(homeWeb3, user.address)
+        } else {
+          retry()
+        }
+      },
+      {
+        retries: 1000000,
```

Comment:
You can use `forever: true` instead of doing this. It's not explicitly documented in `promise-retry`, but since it just sends the options to [retry](https://github.com/tim-kos/node-retry), you can use whichever options are there.